### PR TITLE
Update the message regarding contributing from the codespaces

### PR DESCRIPTION
**Pull request by RotemAmit**
## Related Issues
fixes:

## Description
When contributors contribute content items from an old pack, there is a message in the PR, saying that they contributed from an old version of the pack. In this message, there is an explanation of how to contribute from Codespaces. 
Now, that we have documentation about how to configure and contribute through Codespaces, I updated the message with a link to the relevant page.

### CODE REVIEW

**Review by ShacharKidor — APPROVED** [review]
Nice :)

Please test this change with a PR from MP. 
And let's add a tech doc writer as a reviewer as discussed offline.

**Comment by ShacharKidor** [comment]
> @ShacharKidor I checked the URL and it is correct. So it looks OK. Doc review completed.

@ShirleyDenkberg - Thank you!

**Comment by RotemAmit** [comment]
@ShacharKidor I checked that the message is showing correctly in a PR of contribution to an old pack, [here](https://github.com/demisto/content/pull/29161).

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 * Fixed an issue where the url regex in the **validate** command was wrong.
 * Fixed an issue where **pre-commit** command failed when using global environment.
 * Fixes an issue in **format** command where the `-i` option included files in `.venv` directories.
+* Updated the comment in a contribution to an old pack to contain the link to the documentation about GitHub Codespace.
```
Comment (ShacharKidor) [inline CHANGELOG.md:14]:
```suggestion
* Updated the comment added to contribution PRs to old packs so it contains a link to the documentation of the **GitHub Codespaces** in xsoar.pan.dev.
```

**Review by ShacharKidor — APPROVED** [review]

**Pull request merged**